slmsuite/hardware/slms/slm.py

```diff
@@ -421,8 +431,9 @@ def _phase2gray(self, phase, out=None):
             out -= 1
 
             # This part (along with the choice of type), implements modulo much faster than np.mod().
-            bw = int(self.bitresolution - 1)
-            np.bitwise_and(out, bw, out=out)
+            if self.bitresolution != 8 and self.bitresolution != 16:
+                bw = int(self.bitresolution - 1)
```

Comment:
Can we change `bw` to `active_bits_mask`, `bit_mask` or something more descriptive?